src/flexbox/nasa.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+  <html lang="en">
+  <head>  
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../../src/reset.css">
```

Comment:
same as above:
<link rel="stylesheet" href="../../reset.css">
change to this. reset.css is not in the src folder